extraJars/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
In lieu of adding this new .gitignore file, since this is to just ignore everything in this directory, would it work to instead just add a `extraJars/*` entry to the top-level `.gitignore`?

Reply:
Why a `.gitignore` here? Can't we just add this dir to the top-level `.gitignore`?